src/EduManagementLab.Core/Entities/CourseLineItem.cs

```diff
@@ -6,8 +6,10 @@ namespace EduManagementLab.Core.Entities
     public class CourseLineItem
     {
         public Guid Id { get; set; }
-        public string? Name { get; set; }
-        public string? Description { get; set; }
+        [Required]
+        public string Name { get; set; }
+        [Required]
+        public string Description { get; set; }
```

Comment:
Should description be required?

Reply:
no it shouldn't be required, and it can be null, so user can describe or not, depends on it's own will